graphbolt/src/cache_policy.h

```diff
@@ -134,8 +135,8 @@ struct CacheKey {
   // Access only through an std::atomic_ref instance atomically.
   int8_t reference_count_;
   // Keys are restricted to be 48-bit unsigned integers.
-  uint16_t key_upper_;
-  uint32_t key_lower_;
+  uint16_t key_upper_16_bits_;
```

Comment:
```suggestion
  uint16_t key_first_16_bits_;
```

Reply:
I don't think first and last is more clear than upper and lower. I think the convention is the following:

https://stackoverflow.com/questions/19063210/what-is-lower-and-higher-bits

https://en.wikipedia.org/wiki/Bit_numbering#:~:text=The%20LSb%20is%20sometimes%20referred,bit%20or%20left%2Dmost%20bit.

Reply:
The LSb is sometimes referred to as the low-order bit or right-most bit, due to the convention in [positional notation](https://en.wikipedia.org/wiki/Positional_notation) of writing less significant digits further to the right. The MSb is similarly referred to as the high-order bit or left-most bit.